pkg/engine/engine_test.go

```diff
@@ -633,7 +633,7 @@ func TestLikelyDuplicate(t *testing.T) {
 			name:     "empty strings",
 			val:      chunkSecretKey{"", detectorA.Key},
 			dupes:    map[chunkSecretKey]struct{}{{"", detectorB.Key}: {}},
-			expected: true,
+			expected: false,
```

Comment:
Should empty strings be considered duplicates? I guess it makes sense, but I'd initially tweaked the logic for `likelyDuplicate` to ignore anything smaller than 3 characters.